### ISSUE TITLE
Madmin Devicecontrol Unresponsive for Tap/Swipe

### ISSUE THREAD

**Issue by Deathomen** [Map-A-Droid/MAD#956]
I asked about this a few time in Discord, but nobody else seems to be having this issue or know what to do about it. I also saw the other Devicecontrol issue by Krzee, but this one is not the same.

My Devicecontrol page works 99% as intended. Devices show up and disappear correctly, and the options below the screenshot work just fine. Only issue is that Tap/Swipe does nothing at all - I don't even get the usual "loading" animation as if it's trying to take a new screenshot.

I updated yesterday to latest Master for 64bit_autoconf and also created a Python venv finally (I made sure to run requirements.txt again). Please let me know what info could help troubleshoot this issue.

**Comment by sn0opy** [comment]
Still works for me just fine on Chrome and Firefox. Test another browser and/or clear cache. Also check your browser dev console for any errors.

This does not seem to be an actual bug in MAD.

**Comment by Deathomen** [comment]
Thanks for the suggestion - I had tried clearing cache and using a different browser (IE11), but I don't use Chrome so didn't think to install it. As a test I gave it a shot and it worked just fine.

It turns out this was just a matter of a really old browser version. I've been using a spare computer to flash ATVs and upgrade MAD because my main PC has an AMD CPU that can't flash ATVs. This PC had Firefox 54.0.1, so after upgrading to the latest version 79.0, everything is working fine.

In the non-working scenario, I get a "_Reference Error: "event" is not defined    devicecontrol:455:9_" error message in the browser console. Not that I expect you to do anything, I just figured I would pass along the info in case it was useful. Thanks for the help!
